smartsim/_core/_install/builder.py

```diff
@@ -15,6 +16,25 @@
 #   - check cmake version and use system if possible to avoid conflicts
 
 
+def expand_exe_path(exe):
```

Comment:
While duplicating code is no fun, for the case of things that are used for `setup.py` and all that, it's better to err on the side of keeping it clean. As we discussed briefly, the only thing that I can think of is to import the function in the utils from here and make it available. I say keep it as is, unless @al-rigazzi has any objections.